Multisig/UI/Transactions/TransactionViewModel/TransactionViewModel.swift

```diff
@@ -41,7 +44,10 @@ class TransactionViewModel: Identifiable {
 
     init(_ tx: Transaction, _ safe: SafeStatusRequest.Response) {
         id = UUID()
+        hash = tx.transactionHash?.description
         date = tx.executionDate ?? tx.submissionDate ?? tx.modified
+        createdDate = tx.submissionDate.map { Self.dateFormatter.string(from: $0) } ?? ""
+        executedDate = tx.executionDate.map { Self.dateFormatter.string(from: $0) } ?? ""
```

Comment:
do you really need this to be empty String, because this var is declared as optional?